app/Insightly/Json.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Insightly;
+
+final class Json
```

Comment:
Does Laravel have a class for this by any chance? Or if not, can we move this to just `app` so we can re-use it in other places later and not just the Insightly integration?

Reply:
Laravel doesn't have a class for this. I moved it to the root of the `App` namespace. If we add more JSON-related stuff it could get its own folder.